### PR TITLE
feat(nature-remo): add nature remo extension

### PR DESCRIPTION
**Pull request by uetchy**
Added an extension for [Nature Remo](https://en.nature.global/).

## Screencast

nature-remo-1
nature-remo-2

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by danielsequeira** [comment]
Hi @uetchy 👋 

Thank you so much for your contribution.

I just noticed you are missing a `README.md` file and your changelog should be all in uppercase like `CHANGELOG.md`

**Comment by uetchy** [comment]
@danielsequeira Oh shoot, thanks for the heads up. All fixed now!

File: extensions/nature-remo/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Nature Remo Changelog
+
+## Added Nature Remo (2022-03-15)
```
Comment (danielsequeira) [inline extensions/nature-remo/CHANGELOG.md:3]:
```suggestion
## [Added Nature Remo] - 2022-07-15
```

**Comment by danielsequeira** [comment]
@uetchy one last thing, the icon you added is not really 512x512, which makes it look a lot smaller than other raycast command icons. could you maybe find a 512x512 for this?

**Comment by uetchy** [comment]
@danielsequeira easy-peasy! since I made this icon :)

**Review by danielsequeira — APPROVED** [review]

**Pull request merged**